src/Polly.Shared/Duration/ConstantBackoff.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+
+namespace Polly.Duration
+{
+    /// <summary>
+    /// Generates sleep durations as a constant value.
+    /// The formula used is: Duration = <see cref="Delay"/>.
+    /// For example: 1s, 1s, 1s, 1s.
+    /// </summary>
+    public sealed class ConstantBackoff : ISleepDurationStrategy
+    {
+        /// <summary>
+        /// Whether the first retry will be immediate or not.
+        /// </summary>
+        public bool FastFirst { get; }
+
+        /// <summary>
+        /// The duration value for the first retry.
+        /// </summary>
+        public TimeSpan Delay { get; }
+
+        /// <summary>
+        /// Creates a new instance of the class.
+        /// </summary>
+        /// <param name="delay">The duration value for the first retry.</param>
+        /// <param name="fastFirst">Whether the first retry will be immediate or not.</param>
+        public ConstantBackoff(TimeSpan delay, bool fastFirst = false)
+        {
+            if (delay < TimeSpan.Zero) throw new ArgumentOutOfRangeException(nameof(delay));
+
+            Delay = delay;
+            FastFirst = fastFirst;
+        }
+
+        /// <summary>
+        /// Generate the sequence of <see cref="TimeSpan"/> values to use as sleep-durations.
+        /// </summary>
+        /// <param name="retryCount">The maximum number of retries to use, in addition to the original call.</param>
+        public IReadOnlyList<TimeSpan> Discrete(int retryCount)
+        {
+            if (retryCount < 0) throw new ArgumentOutOfRangeException(nameof(retryCount));
+
+            TimeSpan[] delays = new TimeSpan[retryCount];
+            if (delays.Length == 0)
+                return delays;
+
+            int i = 0;
+            if (FastFirst)
+                delays[i++] = TimeSpan.Zero;
+
+            for (; i < delays.Length; i++)
+            {
+                delays[i] = Delay;
+            }
+
+            return delays;
+        }
+
+        /// <summary>
+        /// Generate a continuous sequence of <see cref="TimeSpan"/> values to use as sleep-durations.
+        /// </summary>
+        /// <param name="retryCount">The maximum number of retries to use, in addition to the original call.</param>
+        public IEnumerable<TimeSpan> Continuous(int retryCount)
```

Comment:
This method is meant to be used in the `Forever()` methods. But not sure if it's adequate or suitable.